<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MacBookAir" failures="3" tests="3" name="ebanking Test(failed)" time="72.446" errors="0" timestamp="2022-07-06T15:35:50 IST">
  <testcase classname="com.ebanking.testCases.TC_LoginDDT_002" name="loginDDT" time="29.031">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.ebanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.ebanking.testCases.TC_LoginDDT_002" name="loginDDT" time="6.193">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.ebanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase classname="com.ebanking.testCases.TC_LoginDDT_002" name="loginDDT" time="6.283">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.ebanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:35)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
</testsuite> <!-- ebanking Test(failed) -->
